msteams-platform/concepts/build-and-test/share-to-teams-overview.md

```diff
@@ -8,11 +8,11 @@ ms.date: 02/09/2023
 
 # Share to Teams
 
-Share to Teams is a button enabled on any web app, personal app, or tab that allows users to share content to other users in Teams.
+Share to Teams is a button enabled on any web app, personal app, or tab that allows users to share content to other users in Teams. You can customize the placement of Share to Teams button on the app. Users can pin the crucial content and refer to it while collaborating with Teams.
 
-You can also decide the placement of the Share to Teams button on the app. Users can pin the crucial content and refer to it while collaborating with Teams.
+In addition to Share to Teams, the **Present Now** button in the meeting stage allows users to present the content for live meetings or Meet Now scenarios. The users can share content and also present it live for real-time collaboration and discussion among Teams users.
 
-:::image type="content" source="../../assets/images/share-to-teams/overview.PNG" alt-text="Screenshot showing the Share button on the app.":::
+:::image type="content" source="~/assets/images/share-to-teams-popup.png" alt-text="Screenshot showing the Share button on the app.":::
```

Comment:
Can we not update the image with present now here - since this option is exclusively for meetings scenario and supported only in web apps scenario and not from personal tabs flow.

Reply:
Fixed the image.